lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -62,75 +66,174 @@ def __init__(self, config):
         self.excluded_features = ('source', 'exon', 'fasta_record')
         self.ont_mappings = load_ontology_mappings('/kb/module/data')
         self.code_table = 11
-        self.re_api_url = config.re_api_url
         # dict with feature 'id's that have been used more than once.
         self.used_twice_identifiers = {}
+
+        # related info for genome process and upload
```

Comment:
Does not matter because these attributes will be set before being used.
But for consistency, I can initiate and assign them to `None` in  _Genome() in the next PR.

Reply:
👍